src/PREfastXmlSarifConverter/ConvertPREfastToSarif.vcxproj

```diff
@@ -93,6 +93,7 @@
       <WarningLevel>Level3</WarningLevel>
       <Optimization>Disabled</Optimization>
       <PreprocessorDefinitions>BUILD_CONSOLE;WIN32;_DEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
```

Comment:
hey, look, it's set there, why isn't it getting honored?

---
In reply to: [115615970](https://github.com/Microsoft/sarif-sdk/pull/564#discussion_r115615970) [](ancestors = 115615970)